pkg/compute/init.go

```diff
@@ -171,6 +169,15 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 		progress = text.NewQuietProgress(out)
 	}
 
+	progress.Step("Checking if git is installed...")
```

Comment:
Don't feel like we need to print this to the output.  Just print the error if it's not there.